docker/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM php:7.1.27-fpm-stretch
```

Comment:
you can simply use `FROM vinelab/nginx-php:7.4` instead of copying it here. Then you can add anything you want to it.

Reply:
nice!... and the building is much faster!